integration/kubernetes/k8s-scale-nginx.bats

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	versions_file="${BATS_TEST_DIRNAME}/../../versions.yaml"
+	nginx_version=$("${GOPATH}/bin/yq" read "$versions_file" "docker_images.nginx.version")
+	nginx_image="nginx:$nginx_version"
+	deployment="nginx-deployment"
+	export KUBECONFIG="$HOME/.kube/config"
+	sudo -E crictl pull "$nginx_image"
+	get_pod_config_dir
+}
+
+@test "Scale nginx deployment" {
+	wait_time=30
+	sleep_time=3
+
+	sed -e "s/\${nginx_version}/${nginx_image}/" \
+		"${pod_config_dir}/${deployment}.yaml" > "${pod_config_dir}/test-${deployment}.yaml"
+
+	kubectl create -f "${pod_config_dir}/test-${deployment}.yaml"
+	kubectl wait --for=condition=Available --timeout=60s deployment/${deployment}
+	kubectl expose deployment/${deployment}
+	kubectl scale deployment/${deployment} --replicas=10
```

Comment:
can you make a variable for the number of replicas? 
Also, I am not sure if our tiny VMs that run the CI will be able to scale to 10 replicas, did you trylocally?

Reply:
@chavafg changes applied, yes I tried it locally with 10 but I reduced the number so we do not have issues on the CI

Reply:
/test